tests/src/io/openliberty/tools/eclipse/test/it/LibertyPluginSWTBotMavenTest.java

```diff
@@ -612,6 +660,92 @@ public void testStartWithDebugAsShortcut() {
         terminal.close();
     }
 
+    @Test
+    public void testStartWithNoWrapperAndNoPreferencesSet() {
+        
+        // verify no wrapper present
+        
+        String localMvnwCmd = LibertyPluginTestUtils.onWindows() ? "mvnw.cmd" : "mvnw";
+        String absoluteMvnwCmd = projectPath.toAbsolutePath().toString() + localMvnwCmd;
+        validateWrapperInProject(false, absoluteMvnwCmd);
+        
+        // verify no preferences are set
+        validateLibertyToolsPreferencesSet(false);
+        
+        // Start dev mode.
+        SWTBotPluginOperations.launchStartWithDashboardAction(bot, dashboard, MVN_APP_NAME);
+
+        // Validate application is not up and not running.
+        validateApplicationOutcome(MVN_APP_NAME, false, projectPath.toAbsolutePath().toString() + "/target/liberty");
```

Comment:
Is there a more direct way to see that we failed to run dev mode than looking for the app not starting?